### PR TITLE
Added autogoto to saved settings

### PR DESCRIPTION
**Pull request by pci06**
Hello,
This is a new PR as ProbePage.py diverge too much
Please see old PR #1701 for reference.

Regards.

### CODE REVIEW

File: bCNC.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 #Autodetect python version
 [ .$PYTHON = . ] && PYTHON=`which python2`
+[ .$PYTHON = . ] && PYTHON=`which python3`
```
Comment (Harvie) [inline bCNC.sh:5]: python2 is not really supported, so python3 should come first. can you please move it one line higher?

**Comment by Harvie** [comment]
other that that. this seems OK.

**Comment by pci06** [comment]
Hello,
python3 is now the preferred interpreter.

**Comment by Harvie** [comment]
thank you

**Pull request merged**